web-console/4.0-designs/storage/storage.md

```diff
@@ -0,0 +1,22 @@
+# Storage
+
+## Create Storage Class
+
+![Storage class page with create button](img/create-storage-class.png)
+- Users with appropriate privileges can click to create a new Storage Class.
```

Comment:
What does it look like when users don't have privileges? Is there an empty state or is the button just disabled?

Reply:
I'm doing a story about RBAC right now and I think any design for this would be the same for every object type, not just storage. I can rephrase this so as not to imply that question though.